Synthetic readings don't calculate

It looks like some of the datasets didn't migrate properly

```
2014-11-11 18:01:15.182 [clojure-agent-send-off-pool-13] INFO  synthetic-readings-q-00 kixi.hecuba.controller.pipeline - Starting synthetic readings job.

2014-11-11 18:01:15.196 [clojure-agent-send-off-pool-13] ERROR synthetic-readings-q-00 kixi.hecuba.storage.db - Query execution failed: SELECT * FROM sensors WHERE sensor_id = null AND device_id = 'b5980a0f00ba365
7862e98865b8102b1643bc8a9'; opts: null
clojure.lang.ExceptionInfo: Query execution failed
        at clojure.core$ex_info.invoke(core.clj:4403) ~[clojure-1.6.0.jar:na]
        at qbits.alia$ex__GT_ex_info.invoke(alia.clj:168) ~[na:na]
        at qbits.alia$ex__GT_ex_info.invoke(alia.clj:170) ~[na:na]
        at qbits.alia$execute.invoke(alia.clj:273) ~[na:na]
        at kixi.hecuba.storage.db.CassandraSession._execute(db.clj:55) ~[na:na]
        at kixi.hecuba.storage.db$execute.doInvoke(db.clj:87) [na:na]
        at clojure.lang.RestFn.invoke(RestFn.java:425) [clojure-1.6.0.jar:na]
        at kixi.hecuba.data.sensors$get_by_id.invoke(sensors.clj:84) [na:na]
        at kixi.hecuba.data.sensors$all_sensor_information.invoke(sensors.clj:307) [na:na]
        at kixi.hecuba.controller.pipeline$build_pipeline$fn__36507$synthetic_readings_q_consumer__36512.invoke(pipeline.clj:258) [na:na]
        at kixi.hecuba.controller.pipeline$build_pipeline$fn__36507.invoke(pipeline.clj:251) [na:na]
        at pipejine.core$consumer$deliver__9263.invoke(core.clj:49) [na:na]
        at pipejine.core$consumer.invoke(core.clj:69) [na:na]
        at pipejine.core$spawn_consumers$fn__9279.invoke(core.clj:91) [na:na]
        at clojure.core$binding_conveyor_fn$fn__4145.invoke(core.clj:1910) [clojure-1.6.0.jar:na]
        at clojure.lang.AFn.call(AFn.java:18) [clojure-1.6.0.jar:na]
        at java.util.concurrent.FutureTask.run(FutureTask.java:262) [na:1.7.0_67]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145) [na:1.7.0_67]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) [na:1.7.0_67]
        at java.lang.Thread.run(Thread.java:745) [na:1.7.0_67]
Caused by: com.datastax.driver.core.exceptions.InvalidQueryException: Invalid null clustering key part sensor_id
        at com.datastax.driver.core.Responses$Error.asException(Responses.java:97) ~[cassandra-driver-core-2.1.0.jar:na]
        at com.datastax.driver.core.DefaultResultSetFuture.onSet(DefaultResultSetFuture.java:110) ~[cassandra-driver-core-2.1.0.jar:na]
```

--- a/src/kixi/hecuba/data/calculate.clj
+++ b/src/kixi/hecuba/data/calculate.clj
@@ -397,9 +397,8 @@
                   (= unit   (:unit %))) sensors)))
 
 (defmethod should-calculate? :divide [ds sensors]
-  (let [{:keys [period unit]} (first sensors)]
-    (every? #(and (= period (:period %))
-                  (= unit (:unit %))) sensors)))
+  (let [{:keys [period]} (first sensors)]
+    (every? #(= period (:period %)) sensors)))
 
 (defmethod should-calculate? :subtract [ds sensors]
   (let [{:keys [period]} (first sensors)]

